src/taco-remote-lib/common/builder.ts

```diff
@@ -162,7 +153,7 @@ class Builder {
         if (xmlVersion) {
             mustReAddPromise = this.getCurrentPlatformVersion().then((currentVersion: string) => {
                 if (!semver.satisfies(currentVersion, xmlVersion)) {
```

Comment:
Oh nevermind This is for re-adding the platform, not whether we need to prepare. Disregard.